ptero_workflow/implementation/models/workflow.py

```diff
@@ -112,10 +113,30 @@ def as_dict(self, detailed):
 
         return result
 
+    def attach_expire_transition(self, transitions, pn, ttl):
+        transitions.append({
+                'inputs': [pn],
+                'action': {
+                    'type': 'expire',
+                    'ttl_seconds': ttl
+                }
+            })
+
     def get_petri_transitions(self):
         transitions = []
         success_place, failure_place = self.root_task.attach_transitions(
                 transitions, self.start_place_name)
+
+        success_ttl = os.environ.get('PTERO_WORKFLOW_SUCCESS_EXPIRE_SECONDS')
```

Comment:
I would prefer if these used the same names as the statuses that the workflow could end up in.  i.e. succeeded and failed